recipe/meta.yaml

```diff
@@ -39,7 +39,7 @@ test:
     - test
   requires:
     - pytest
-    - shapely  # [not osx or py<=39]
+    - shapely  # [(not osx or py<=39) and python_impl != 'pypy']
```

Comment:
This is a bit confusing, but we want to skip OSX on Python 3.10 for reasons I don't recall and PyPy.